prj.conf

```diff
@@ -60,6 +64,9 @@ CONFIG_MBEDTLS_SSL_MAX_CONTENT_LEN=4096
 CONFIG_MBEDTLS_MEMORY_DEBUG=y
 CONFIG_MBEDTLS_SHELL=y
 
+CONFIG_BASE64=y
+CONFIG_TLS_CREDENTIALS_SHELL=y
```

Comment:
Neat! I feel like one of these days I need to look at all the libraries with shells and keep a few in my back pocket